x/wasm/keeper/migrations.go

```diff
@@ -25,3 +27,27 @@ func (m Migrator) Migrate1to2(ctx sdk.Context) error {
 	})
 	return nil
 }
+
+// Migrate2to3 migrates the x/wasm module state from the consensus
+// version 2 to version 3. Specifically, it takes the parameters that are currently stored
+// and managed by the x/params module and stores them directly into the x/wasm
+// module state.
+func (m Migrator) Migrate2to3(ctx sdk.Context) error {
```

Comment:
Good 👁️ 
Should we move migrations to a new package as the SDK [did](https://github.com/cosmos/cosmos-sdk/tree/release/v0.47.x/x/staking/migrations)?